app/src/main/java/com/example/mindspace/ui/screens/AuthScreen.kt

```
@@ -0,0 +1,393 @@
+                            )
+                        }
+
+                        GradientOutlinedButton(
+                            onClick = {
+                                oneTapClient.beginSignIn(signInRequest)
+                                    .addOnSuccessListener { result ->
+                                        try {
+                                            launcher.launch(
+                                                IntentSenderRequest.Builder(result.pendingIntent).build()
+                                            )
+                                        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)